spec/rspec/core/example_group_spec.rb

```diff
@@ -1366,12 +1366,12 @@ def extract_execution_results(group)
     describe "ivars are not shared across examples" do
       it "(first example)" do
         @a = 1
-        expect(defined?(@b)).to be_falsey
+        expect(defined?(@b)).to be(nil)
       end
 
       it "(second example)" do
         @b = 2
-        expect(defined?(@a)).to be_falsey
+        expect(defined?(@a)).to be(nil)
```

Comment:
I've changed one of the examples above to `expect(defined?(...)).to be`.
WDYT of using `not_to be` here?

Reply:
`.to be_nil` you mean? Yeah, `not_to be_nil` is fine for `defined?` I imagine

Reply:
Oh, didn't know you could simply write `to be`. Cool, Shakespeare would love it.